### ISSUE TITLE
bazel 3.4.0 fails to run in docker during rbe_autoconfig

### ISSUE THREAD

**Issue by shahms** [bazelbuild/bazel-toolchains#899]
I'm not sure if this belongs here or in the Bazel repo itself, but given WORKSPACE:
```
http_archive(
    name = "bazel_toolchains",
    sha256 = "882fecfc88d3dc528f5c5681d95d730e213e39099abff2e637688a91a9619395",
    strip_prefix = "bazel-toolchains-3.4.0",
    urls = [
        "https://github.com/bazelbuild/bazel-toolchains/releases/download/3.4.0/bazel-toolchains-3.4.0.tar.gz",
        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/releases/download/3.4.0/bazel-toolchains-3.4.0.tar.gz",
    ],
)
load("@bazel_toolchains//rules:environments.bzl", "clang_env")
load("@bazel_toolchains//rules:rbe_repo.bzl", "rbe_autoconfig")
load("//tools/platforms:toolchain_config_suite_spec.bzl", "DEFAULT_TOOLCHAIN_CONFIG_SUITE_SPEC")

rbe_autoconfig(
    name = "rbe_default",
    env = clang_env(),
    export_configs = True,
    toolchain_config_suite_spec = DEFAULT_TOOLCHAIN_CONFIG_SUITE_SPEC,
    use_legacy_platform_definition = False,
)
```
Attempting to rebuild the RBE config with .bazelversion == 3.4.0 times out connecting to the server after 2 minutes.
Debugging from a command line in the container that gets created shows:
```
( /usr/bin/docker run -it --entrypoint 'bash' --env ABI_LIBC_VERSION=glibc_2.19 --env ABI_VERSION=clang --env BAZEL_COMPILER=clang --env BAZEL_HOST_SYSTEM=i686-unknown-linux-gnu --env BAZEL_TARGET_CPU=k8 --env BAZEL_TARGET_LIBC=glibc_2.19 -
-env BAZEL_TARGET_SYSTEM=x86_64-unknown-linux-gnu --env CC=clang --env CC_TOOLCHAIN_NAME=linux_gnu_x86 --env USE_BAZEL_VERSION=3.4.0 --user root --volumes-from 1b74da4122c98fe34501a8fbe5682fc4f97d708f9e2b968b63a7fb4c0f34210c gcr.io/kythe-repo
/kythe-builder@sha256:affd4b4d05144cb524db0d4f4639db74942b53ff499a1dfce23ce0592654e259 )
root@72b200d978a1:/# cd /rbe_autoconf/project_src
root@72b200d978a1:/rbe_autoconf/project_src# /rbe_autoconf/bazelisk/bazelisk build @local_config_cc//...
Killed non-responsive server process (pid=2405)
Starting local Bazel server and connecting to it...
... still trying to connect to local Bazel server after 10 seconds ...
... still trying to connect to local Bazel server after 20 seconds ...
... still trying to connect to local Bazel server after 30 seconds ...
... still trying to connect to local Bazel server after 40 seconds ...
... still trying to connect to local Bazel server after 50 seconds ...
... still trying to connect to local Bazel server after 60 seconds ...
... still trying to connect to local Bazel server after 70 seconds ...
... still trying to connect to local Bazel server after 80 seconds ...
... still trying to connect to local Bazel server after 90 seconds ...
... still trying to connect to local Bazel server after 100 seconds ...
... still trying to connect to local Bazel server after 110 seconds ...
FATAL: couldn't connect to server (7228) after 120 seconds.
root@72b200d978a1:/rbe_autoconf/project_src# USE_BAZEL_VERSION=3.3.1 /rbe_autoconf/bazelisk/bazelisk build @local_config_cc//...
Killed non-responsive server process (pid=7228)
Starting local Bazel server and connecting to it...
INFO: Analyzed 9 targets (13 packages loaded, 50 targets configured).
INFO: Found 9 targets...
INFO: Deleting stale sandbox base /root/.cache/bazel/_bazel_root/bd8f6376282b6bf2b9adeeab81e3922e/sandbox
INFO: Elapsed time: 2.134s, Critical Path: 0.03s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
```

**Comment by katre** [comment]
See https://github.com/bazelbuild/bazel/issues/11756, it's an issue int he gRPC update and we've released 3.4.1 to mitigate this.

**Comment by shahms** [comment]
Thanks!
